.ci/releasePipeline.groovy

```diff
@@ -141,3 +134,175 @@ pipeline {
         }
     }
 }
+
+
+String getVersionFromGitTagName() {
+    def ver = sh(returnStdout: true, script: "git tag -l --points-at HEAD --sort=-creatordate | head -n 1")
+        .trim()
+        .substring(1)  // drop v prefix
+    return ver
+}
+
+void getSignClient() {
+    withVault(vaultSecrets: [[path        : 'keptn-jenkins/monaco/sign-service',
+                              secretValues: [[envVar: 'repository', vaultKey: 'client_repository_url ', isRequired: true]]]]
+    ) {
+        if (!fileExists('sign-client.jar')) {
+            sh '''
+                curl --request GET $repository/testautomation-release-local/com/dynatrace/services/signservice/client/1.0.[RELEASE]/client-1.0.[RELEASE].jar
+                     --show-error
+                     --silent
+                     --globoff
+                     --fail-with-body
+                     --output sign-client.jar
+               '''.replaceAll("\n", " ").replaceAll(/ +/, " ")
+        }
+    }
+}
+
+void buildBinary(Map args = [command: null, version: null, dest: null]) {
+    stage('🏁 Build ' + args.command) {
+        sh "make ${args.command} VERSION=${args.version} OUTPUT=${args.dest}"
+    }
+}
+
+void signWinBinaries(Map args = [source: null, version: null, destDir: null, projectName: null]) {
+    stage('Sign binaries') {
+        signWithSignService(source: args.source, version: args.version, destDir: args.destDir, projectName: args.projectName, signAction: "SIGN")
+    }
+}
+
+void createShaSum(Map args = [source: null, dest: null]) {
+    stage('Compute SHA sum') {
+        sh "sha256sum ${args.source} | sed 's, .*/, ,' > ${args.dest}"
```

Comment:
```suggestion
        sh "sha256sum -z ${args.source} | sed 's, .*/, ,' > ${args.dest}"
```

I just tried this for the linux-amd sum from the 0.7 release, and with the envelope still results in new-lines in the sum getting windows new-line characters. 
That results in a shasum file that's invalid, as it contains the windows (**0xD** 0xA) line-ending rather than just 0xA. (0xD == ^M)

Simply not having a newline in the shasum file makes this work.

```
8bbea02452ecc552ded0d5a4000fd71334e380ad2b75b8a64dc2721f6a6dd61e monaco-linux-amd64^M
```

Reply:
done